app/example/completion.usage

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+echo -e "\033[36mTODO\033[39m: Implement this command"
```

Comment:
Could I suggest returning an empty completions list here instead? Printing during completion generation can sometimes be a bit "weird" and ideally I'd like the example to mostly work "out of the box" for people who don't customize it.

If you think it's worth exposing a flag, you could maybe create a "fake" completion suggestion like `edit-completions-file-to-add-completion-support`?

```suggestion
OPTIONS=("edit-completions-file-to-add-completion-support")
echo ${OPTIONS[@]}
```

Reply:
I might misunderstood your request but I added that to `app/examples/completion.complete` file:
```sh
OPTIONS=("one" "two" "three")
echo ${OPTIONS[@]}
```
`[command].complete` file isn't being created on `command create`. User supposed to create it manually, as in my case a lot of commands don't need to have suggestions.

Do you want me to add a code to create default (empty?) `.complete` file each time?

Reply:
Ah, gotcha - in that case please don't worry (unless you think that will be useful to add). Thanks for adding this!